geomesa-features/geomesa-feature-kryo/src/main/scala/org/locationtech/geomesa/features/kryo/impl/KryoFeatureDeserializationV2.scala

```diff
@@ -24,6 +24,18 @@ import org.opengis.feature.simple.SimpleFeatureType
 
 import scala.util.control.NonFatal
 
+/**
+  * Version 2 serialization scheme consists of:
```

Comment:
Can new SFTs select the v2 Kryo format?

Reply:
no... we could make that an option i guess. ideally there'd be no reason to